client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilter.tsx

```diff
@@ -49,6 +49,6 @@ export default DeskFilter;
 
 interface Props {
     desks: Desk[];
-    filteredDesks: number[];
+    filteredDesks: (number | null)[];
     onFilterChange: (event: React.ChangeEvent<{}>, selectedDesks: Desk[]) => void;
```

Comment:
Why can the desk filter be null? If it's empty shouldn't it be [ ]?

Reply:
desks filter is the filter by the desk id, due to changing the desk id on unassigned desk to be null, this means that it can be nullable